Class-Code/Elementary-CA/ghouston/sketch.js

```diff
@@ -17,13 +17,22 @@ function draw() {
   background(120);
 
   for (let i = 0; i < gen1.length; i++) {
-    if (gen1(i) == 1) {
+    if (gen1[i] == 1) {
       fill(0);
     } else {
       fill(255);
     }
-      noStroke()
-      rect(i * gridsize, 0, gridsize, gridsize);
-    }
+    noStroke();
+    rect(i * gridsize, 0, gridsize, gridsize);
   }
 }
+
+//generate
```

Comment:
This looks like a good start, but you need to make sure that all of this goes into a function
```
function generate(){
    .
    .
    .
    ca.push(nextGen)
}